Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.ipp

```diff
@@ -282,7 +280,17 @@ Acts::Result<void> Acts::
           }
           trkAtVtx.linearizedState = *result;
           trkAtVtx.isLinearized = true;
+        } else if (state.vtxInfoMap[vtx].relinearize) {
+          auto result = linearizer.linearizeTrack(
+              m_extractParameters(*trk), state.vtxInfoMap[vtx].oldPosition,
+              vertexingOptions.geoContext, vertexingOptions.magFieldContext,
+              state.linearizerState);
+          if (!result.ok()) {
+            return result.error();
+          }
           state.vtxInfoMap[vtx].linPoint = state.vtxInfoMap[vtx].oldPosition;
```

Comment:
Instead of doubling the code with an if else wouldn't it be simpler to add an if before the `state.vtxInfoMap[vtx].linPoint = state.vtxInfoMap[vtx].oldPosition;` ?